### ISSUE TITLE
Feature: second "bloodsugar" value

### ISSUE THREAD

**Issue by NotSoImportant** [Faltenreich/Diaguard#15]
Be able to enter a value for e.g. flash glukose.
Either as a new type or a second input for bloodsugar.

**Comment by Faltenreich** [comment]
I am not sure whether I get your suggestion right, so I will try to phrase it myself which than can be accepted or corrected by you again:

You need an option to enter two blood sugar values at the same time because they may deviate from each other due to different technologies being used, e.g. when measuring with a CGM/FGM and additionally with a normal blood sugar measuring device.

**Comment by NotSoImportant** [comment]
Exactly. I want to track both of them because they might deviate or because I want to be able to track them separately.
